api/api.go

```diff
@@ -2,10 +2,14 @@ package api
 
 import (
 	"github.com/gin-gonic/gin"
+	"github.com/trustwallet/blockatlas/api/endpoint"
+	"github.com/trustwallet/blockatlas/pkg/servicerepo"
 	"github.com/trustwallet/blockatlas/platform"
 )
 
-func SetupPlatformAPI(root gin.IRouter) {
+func SetupPlatformAPI(serviceRepo *servicerepo.ServiceRepo, root gin.IRouter) {
+	endpoint.Init(serviceRepo)
```

Comment:
We need to init all the services at 1 place

Reply:
done